src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -57,6 +63,11 @@ internal TagHelperExecutionContext(string tagName)
         }
 
         /// <summary>
+        /// Indicates whether or not the tag of the HTML element this context is for self-closing.
```

Comment:
nits: `/// Gets a value indicating whether or not the tag in the Razor source was self-closing.`